### ISSUE TITLE
Server ticrate Issues

### ISSUE THREAD

**Issue by yuljk** [dreamstalker/rehlds#615]
Hi guys - I'm running 3.4.0.651-dev.  I've noticed that when checking 'stats' in rcon that my server never goes above 100FPS - I have sys_ticrate 500 set in my launch options.  

On the official HLDS version I was achieving 400-500 FPS.

Is this by design that it is lower, or is there an issue with this build?

Server running on 2016 Server VM with 4GB RAM and 2 vCPU's (3Ghz quad cores)

Many thanks

**Comment by hashnimmo** [comment]
Same here. My server: Ubuntu 18.04 LTS, 1 Core, 512MB RAM (Vultr $2.5/mo plan)

Just mentioning this because in-case if this is somehow not normal, other than that there are no stability problems in the server or anything. Just reporting for information and appreciate that ReHLDS seems to be showing the actual server FPS than the old HLDS unstable/fake FPS. So that's a plus.

**Comment by yuljk** [comment]
Yeah - If this is by design fair enough.   Will see what the developers say regarding this

**Comment by OsweRRR** [comment]
the fps are considered false after 1000

**Comment by regener** [comment]
On 3.4.0.654-dev I have 450+ currently.
Are you sure that 512MB RAM is enough?

With 68 plugins (3 of them with SQL usage and 1 with file-write stuff) uses ~1250MB of ram for me.
